frameworks/FSharp/giraffe/src/App/Program.fs

```diff
@@ -0,0 +1,62 @@
+module App.App
+
+open Microsoft.AspNetCore.Hosting
+open Giraffe
+
+[<CLIMutable>] 

```

Comment:
Mhm. Not 100 percent sure, but usually we don't need cli mutable. I don't have it anywhere in any of my giraffe apps